home/tests.py

```diff
@@ -193,3 +184,24 @@ def test_response_context(self, response):
                 ]
             for check, excepted in expectedPairs:
                 assert str(response.context[check]).startswith(excepted)
+
+
+class TestTagsPage:
+    @pytest.mark.django_db
+    def test_tags_page_url(self, tags_response):
+        assert tags_response.status_code == 200
+
+    @pytest.mark.django_db
+    def test_tags_page_context(self, tags_response):
+        excepted = '<QuerySet [<Tag: 5th_Grade>, <Tag: Bagrut_Exam>, <Tag: Hebrew>, <Tag: Pitagoras>]>'
+        assert str(tags_response.context['tags']) == excepted
```

Comment:
its usually a bad idea to use this kind of single-string comparison when you have a more complex data structure. Consider that you might change `__repr__` for various reasons in the future and break the test.

What you should do instead it break down the elements ou want to test - wanna test the type of the object youve got? Use `isinstance`. Wanna test that you got a list of a particular data items? Convert is to a list of simpler objects, like I did in the testing examples I wrote for the message board.